modules/miscutil/lib/inveniocfg.py

```diff
@@ -918,7 +918,7 @@ def cli_cmd_create_apache_conf(conf):
         AliasMatch /sitemap-(.*) %(webdir)s/sitemap-$1
         Alias /robots.txt %(webdir)s/robots.txt
         Alias /favicon.ico %(webdir)s/favicon.ico
-        WSGIDaemonProcess invenio processes=5 threads=1 display-name=%%{GROUP} inactivity-timeout=3600 maximum-requests=10000
+        WSGIDaemonProcess invenio processes=5 threads=1 display-name=%%{GROUP} inactivity-timeout=3600 maximum-requests=10000 %(wsgiuser)s
```

Comment:
I didn't realize that `wsgiuser` was already prefixed with `user=`. Should be good in that case!